dev/TreeView/TreeView.cpp

```diff
@@ -254,7 +254,7 @@ void TreeView::SyncRootNodesWithItemsSource()
     }
 
     auto children = winrt::get_self<TreeViewNodeVector>(RootNodes());
-    children->ClearCore();
+    children->Clear();
```

Comment:
>Clear [](start = 14, length = 5)

Should this clear(false)? I don't expect customer we need to update itemssource.